src/modules/micrortps_bridge/micrortps_client/CMakeLists.txt

```diff
@@ -59,18 +58,34 @@ if(NOT GENERATE_RTPS_BRIDGE MATCHES "off")
         list(APPEND topic_bridge_files_out ${msg_out_path}/${topic}_Subscriber.h)
     endforeach()
 
-    add_custom_command(OUTPUT ${topic_bridge_files_out}
-        COMMAND ${PYTHON_EXECUTABLE}
-        ${PX4_SOURCE_DIR}/Tools/generate_microRTPS_bridge.py
-        -f $ENV{FASTRTPSGEN_DIR}
-        -s ${send_topic_files}
-        -r ${receive_topic_files}
-        -t ${topic_msg_path}
-        -u ${msg_out_path}
-        DEPENDS ${DEPENDS} ${send_topic_files} ${receive_topic_files}
-        COMMENT "Generating RTPS topic bridge"
-        VERBATIM
-        )
+    if (NOT "${config_rtps_send_topics}" STREQUAL "" OR NOT "${config_rtps_receive_topics}" STREQUAL "")
```

Comment:
I'm not sure if that detects if the variable is defined vs the variable is empty